cassandra/src/main/java/se/tre/freki/storage/cassandra/search/CassandraSearchPlugin.java

```diff
@@ -140,11 +237,11 @@ public void onSuccess(final ResultSet result) {
 
         @Override
         public void onFailure(final Throwable throwable) {
-          LOG.error("Write of ngram {} with id {} of type {}", nGram, labelId, type );
+          LOG.error("Write of ngram {} with id {} of type {}", nGram, labelId, type);
```

Comment:
Can you add the word `failed` to the end of the log message and also log the throwable (add it to the end of the call without an interpolation symbol in the message).